### ISSUE TITLE
Access Levels

### ISSUE THREAD

**Issue by ankurparihar** [jdalrymple/gitbeaker#3500]
**Description**

Hi

I would like to use `minAccessLevel` option in my calls. Gitlab has access levels defined as number (Please refer [here](https://docs.gitlab.com/ee/api/members.html#valid-access-levels))
Do we have any way to pass this as a predefined variable instead of numeric value?

**Current**
```js
const groups = await client.Groups.all({minAccessLevel: 40});
```

**Proposal**
```js
const { AccessLevels } = require('@gitbeaker/rest');
const groups = await client.Groups.all({minAccessLevel: AccessLevels.MAINTAINER});
```

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
Yup, I think i would just need to export the enum!

**Comment by ankurparihar** [comment]
Thank you! Let me know if I can help.

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.28.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.28.0) :rocket:

**Comment by ankurparihar** [comment]
Thank you @jdalrymple
